src/app.ts

```diff
@@ -30,7 +30,8 @@ program
   .command('build')
   .description('Builds the report into a folder named "dist"')
   .action(() => {
-    Builder.create().build().catch(err => {
+    const cliConfig = loadCliConfig();
```

Comment:
Like this, it would be really easy to adapt the interface of the CLI so that the `distPath` can, for instance, be given as an option. But the logic itself remains untouched 👏